### ISSUE TITLE
Some questions about the fixed HR classifier in the pretrain.py

### ISSUE THREAD

**Issue by Najlepszy** [ermongroup/PatchDrop#2]
I doubt if only use  `rnet.eval().cuda() # HR Classifier is Fixed` can fix the rnet during training mode...I'm not sure whether rnet is fixed and its parameters do not update while training. Should we use torch.no_grad() or require_grads=False here?
https://github.com/ermongroup/PatchDrop/blob/5d1da64d7ff00774be5af5c398c71614fdd92792/pretrain.py#L197

**Comment by uzkent** [comment]
I do not think we need to do `torch.no_grad` explicitly. One way to check it is to compute the accuracy of the HR classifier on the original images before and after pre-training. On my end, it stays the same. Let me know if you see any change.

**Comment by uzkent** [comment]
I am closing this issue. Let me know if you have any question.

**Comment by uzkent** [comment]
I am closing this issue. Let me know if you have any question.

**Comment by uzkent** [comment]
I am closing this issue. Let me know if you have any question.

**Comment by uzkent** [comment]
I am closing this issue. Let me know if you have any question.
